jupyterhub/apihandlers/users.py

```diff
@@ -244,7 +259,7 @@ async def delete(self, name):
 
     @admin_only
     async def patch(self, name):
-        user = self.find_user(name)
```

Comment:
Do we really need the full User object here?

Reply:
Is there a downside to consistency? In nearly all cases, the user is in memory already, and this adds only a single dict lookup to return the object.